tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/SparkTestBase.scala

```diff
@@ -32,6 +32,19 @@ import org.apache.celeborn.service.deploy.MiniClusterFeature
 
 trait SparkTestBase extends AnyFunSuite
   with Logging with MiniClusterFeature with BeforeAndAfterAll with BeforeAndAfterEach {
+
+  val sparkConf = new SparkConf().setAppName("rss-demo").setMaster("local[2,3]")
+  val sparkSession = SparkSession.builder().config(sparkConf).getOrCreate()
+  val version =
+    try {
+      sparkSession.sql("select version()").collect()(0).getString(0)
+    } catch {
+      case _: Exception => ""
+    }
+  sparkSession.close()
+  val Spark3OrNewer = version >= "3.0"
```

Comment:
it's fragile, and before Spark 3.0, `select version()` returns Hive client version

Reply:
Is other ways to determine the spark version?

Reply:
[SparkSession.version](https://spark.apache.org/docs/latest/api/scala/org/apache/spark/sql/SparkSession.html#version:String)

Reply:
Why not directly use `SparkVersionUtil#isGreaterThan`?